.github/workflows/test.yml

```diff
@@ -17,13 +17,13 @@ jobs:
     # needs: lint
     strategy:
       matrix:
-        ckan-version: [2.8]
+        ckan-version: [2.9]
       fail-fast: false
 
     name: CKAN ${{ matrix.ckan-version }}
     runs-on: ubuntu-20.04
     container:
-      image: openknowledge/ckan-dev:${{ matrix.ckan-version }}
+      image: openknowledge/ckan-dev:2.8
```

Comment:
add a comment here about why we need to use the 2.8 version of the image?

Reply:
Comment added.